src/pages/Categories.jsx

```diff
@@ -0,0 +1,9 @@
+const Categories = () => {
+  return (
+    <div className="p-4">
+      <button className="border-2 border-black px-2 py-1">check status</button>
+    </div>
+  );
+};
```

Comment:
This can be improved by doing a direct return

Reply:
Delete the return keywords and wrap jsx with bracket so as to use the arrow function direct return